lib/merge_hris_client/api/bank_info_api.rb

```diff
@@ -94,7 +90,6 @@ def bank_info_list_with_http_info(x_account_token, opts = {})
       query_params[:'created_after'] = opts[:'created_after'] if !opts[:'created_after'].nil?
       query_params[:'created_before'] = opts[:'created_before'] if !opts[:'created_before'].nil?
       query_params[:'cursor'] = opts[:'cursor'] if !opts[:'cursor'].nil?
-      query_params[:'employee'] = opts[:'employee'] if !opts[:'employee'].nil?
```

Comment:
can we re-add employee here? I still need to look into why our spec is excluding it now

Reply:
done